tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -75,3 +75,13 @@ Compose Up with link
     Should Be Equal As Integers  ${rc}  0
     Should Contain  ${output}  PING aaa
     Should Not Contain  ${output}  bad address 'aaa'
+
+Compose bundle creation
+    ${rc}  Run And Return Rc  docker-compose %{VCH-PARAMS} --file basic-compose.yml down
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  Run And Return Rc  docker-compose %{VCH-PARAMS} --file basic-compose.yml pull
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{VCH-PARAMS} --file basic-compose.yml bundle
+    Log  ${output}
+    Should Contain ${output} Wrote bundle
```

Comment:
I thought so too, but I see a lot of tests without double whitespaces.  I'll add them just in case.